arcee/api.py

```diff
@@ -292,6 +292,9 @@ def start_alignment(
     pretrained_model: Optional[str] = None,
     merging_model: Optional[str] = None,
     alignment_model: Optional[str] = None,
+    hf_model: Optional[str] = None,
+    target_compute: Optional[str] = None,
+    capacity_id: Optional[str] = None,
```

Comment:
Adding a brief description for each of these args could be helpful and make the function definition structure more consistent.